conf/distro/include/rdk-bpi.inc

```diff
@@ -32,3 +32,5 @@ DISTRO_FEATURES_append = " rdkb_cellular_manager_mm"
 
 #Dac Feature support for BPIR4 device.
 DISTRO_FEATURES_append = " dac"
+
+DISTRO_FEATURES_append = " partner_default_ext"
```

Comment:
do we have similar distro in rpi

Reply:
yes chandra..
Please see the below link for your reference,
https://code.rdkcentral.com/r/plugins/gitiles/rdk/components/generic/rdk-oe/meta-cmf-raspberrypi/+/refs/heads/rdk-next/conf/distro/include/rdk-rpi.inc#204